DEVELOPMENT.md

```diff
@@ -3,14 +3,27 @@
 * [Java 21](https://openjdk.org/install)
 * [Docker](https://docs.docker.com/engine/install)
 * [Node.js 18](https://nodejs.org/en/download) (or higher)
+* [Google Chrome](https://www.google.com/chrome/)
 * [Intellij Idea](https://www.jetbrains.com/help/idea/installation-guide.html) (optional)
 
-## Clone this repo
+### On Linux
```

Comment:
I don't think you need any of this for development. Why do you think we need these?

Reply:
Google Chrome is required, as cypress is configured for it.

Reply:
Oki